src/sparseml/exporters/transforms/flatten_qparams.py

```diff
@@ -48,11 +52,12 @@ def transform(self, model: ModelProto) -> ModelProto:
         for init in model.graph.initializer:
             if init.name not in inits_to_flatten:
                 continue
+            _LOGGER.debug(f"Flattening initializer {init.name}")
             a = numpy_helper.to_array(init)
             assert a.shape == (1,)
             b = numpy.array(a[0])
             assert b.shape == ()
             assert b.dtype == a.dtype
             init.CopyFrom(numpy_helper.from_array(b, name=init.name))
-
+        _LOGGER.info(f"Flattened {len(inits_to_flatten)} initializers")
```

Comment:
whats the reasoning for info logging all the minor transforms as opposed to just the main QDQ conversions? not sure I would care about this outside of debug and might prefer to just parse through the main transforms' logs

Reply:
Consistency mainly, I'm fine with debug for the non-core ones